src/IRac.cpp

```diff
@@ -151,6 +151,9 @@ bool IRac::isProtocolSupported(const decode_type_t protocol) {
 #if SEND_MITSUBISHI136
     case decode_type_t::MITSUBISHI136:
 #endif
+#if SEND_MITSUBISHI112
+    case decode_type_t::MITSUBISHI112:
+#endif
```

Comment:
Minor nitpick: Please move it above 136, to maintain the (rather poor :-) sorted order. ;-)